src/services/services.ts

```diff
@@ -1362,7 +1363,7 @@ class SyntaxTreeCache {
                 ),
                 setExternalModuleIndicator: getSetExternalModuleIndicator(this.host.getCompilationSettings()),
             };
-            sourceFile = createLanguageServiceSourceFile(fileName, scriptSnapshot, options, version, /*setNodeParents*/ true, scriptKind);
+            sourceFile = createLanguageServiceSourceFile(fileName, scriptSnapshot, options, version, /*setNodeParents*/ true, scriptKind, this.host.jsDocParsingMode);

```

Comment:
Should this be "SkipAll" because this is syntax only file?

Reply:
Good point, I think it may be able to!

Reply:
Hm, it fails tests; I remember now that we have syntactic classifications for JSDoc nodes, so that would break by default which we don't want.

Reply:
So that being said, I think it's best to leave this as in the PR; I thought about reverting it back to the default (e.g. skip node), but my feeling is that anyone who configures this downstream will understand the implications. They can expect many other things to also be broken if they skip more than they need to function.

Reply:
I feel like we should leave this non configurable for now. And remove LS.jsDocParsingMode as its not used

Reply:
I guess I could remove it; I can actually already make it non-configurable by marking `jsDocParsingMode` internal on `LanguageServiceHost` (this thread turned out to be redundant with the other one). Which would you prefer?

Reply:
We should make this non configurable and use whichever value is correct here.

Reply:
Alrighty, I'll drop it!